### PR TITLE
Move `useMergeRefs` and removes overflow from storybook decorator.

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes

These changes are from #2546. `Table` will be pre-released, but these changes should have a regular release.

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

File: .changeset/polite-kids-admire.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/hooks': patch
+---
+
+Adds `useMergeRefs` into `useMergeRef` directory
```
Comment (TheSonOfThomp) [inline .changeset/polite-kids-admire.md:5]: No actual changes to the hook, right? Just moving the file & tests?

File: .changeset/nice-tips-hide.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@lg-tools/storybook-decorators': patch
+---
+
+Removes `overflow: auto` from `rootStyles`.
```
Comment (TheSonOfThomp) [inline .changeset/nice-tips-hide.md:5]: This prevents automatic scrolling?

**Review by TheSonOfThomp — APPROVED** [review]

File: .changeset/polite-kids-admire.md (hunk 1 above)
Comment (shaneeza) [reply .changeset/polite-kids-admire.md:5]: no changes

File: .changeset/nice-tips-hide.md (hunk 2 above)
Comment (shaneeza) [reply .changeset/nice-tips-hide.md:5]: This was preventing position sticky from working. I don't think this breaks anything else.

**Pull request merged**